<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssistantDialog</class>
 <widget class="QWidget" name="AssistantDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>480</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QLabel" name="explanatoryTextLabel">
     <property name="text">
      <string notr="true">[ explanatory text ] this is a placeholder this is a placeholder this is a placeholder this is a placeholder this is a placeholder this is a placeholder this is a placeholder this is a placeholder this is a placeholder this is a placeholder.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="justCopyButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string notr="true">Copy/Move placeholder</string>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="description">
      <string notr="true">&lt;placeholder text&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Transcode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QListWidget" name="formatListWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="uniformItemSizes">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>3</number>
            </property>
            <item>
             <widget class="QLabel" name="formatIconLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="text">
               <string notr="true"/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>10</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="formatNameLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string notr="true">Format Name</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Transcoding::OptionsStackedWidget" name="transcodingOptionsStackedWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="transcodeAllRadioButton">
        <property name="toolTip">
         <string>Transcode all tracks to the selected format</string>
        </property>
        <property name="whatsThis">
         <string>Transcode all tracks to the selected format</string>
        </property>
        <property name="text">
         <string>Transcode all tracks</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="transcodeOnlyIfNeededRadioButton">
        <property name="toolTip">
         <string>Transcode tracks only when needed for playability in the destination collection</string>
        </property>
        <property name="whatsThis">
         <string>Transcode tracks only when needed for playability in the destination collection</string>
        </property>
        <property name="text">
         <string>Transcode only when needed for playability</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="transcodeUnlessSameTypeRadioButton">
        <property name="toolTip">
         <string>Transcode only when source and destination file formats different</string>
        </property>
        <property name="whatsThis">
         <string>Transcode only when source and destination file formats different</string>
        </property>
        <property name="text">
         <string>Transcode only when source and destination formats are different</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="encoderNotFoundLabel">
     <property name="text">
      <string>&lt;b&gt;Note:&lt;/b&gt; No encoder is available. If you want to transcode tracks please install &lt;i&gt;ffmpeg&lt;/i&gt; or &lt;i&gt;libav&lt;/i&gt; package (with &lt;i&gt;ffmpeg&lt;/i&gt; wrapper) with appropriate encoders. Otherwise you may check &lt;i&gt;Remember this choice for the next time&lt;/i&gt; option in order to skip this dialog for future transfers.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="rememberCheckBox">
     <property name="text">
      <string>Remember this choice for the next time</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Transcoding::OptionsStackedWidget</class>
   <extends>QWidget</extends>
   <header>transcoding/TranscodingOptionsStackedWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
